x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.test.ts

```diff
@@ -351,4 +351,28 @@ describe('ruleParamsModifier', () => {
       expect(editedRuleParams.timelineTitle).toBe('Test timeline');
     });
   });
+
+  describe('schedule', () => {
+    test('should set timeline', () => {
```

Comment:
`should set timeline` is probably a copy-paste

Reply:
Fixing in follow up PR